test/guttering_systems_test.cpp

```diff
@@ -118,12 +118,28 @@ static void run_test(const int nodes, const int num_updates, const int data_work
   for (int t = 0; t < data_workers; t++)
     query_threads[t] = std::thread(querier, gts, nodes);
 
-  for (int i = 0; i < num_updates; i++) {
-    update_t upd;
-    upd.first = i % nodes;
-    upd.second = (nodes - 1) - (i % nodes);
-    gts->insert(upd);
-  }
+  // In case there are multiple threads
+  std::vector<std::thread> threads;
+  threads.reserve(nthreads);
+  // This is the work to do per thread (rounded up)
+  const int work_per = (num_updates+nthreads-1) / nthreads;
+
+  auto task = [&](const int j){
```

Comment:
Use separate tests for this. The idea of this file is to have all the tests for both guttering systems living together. A couple tests for multithreading the standalone gutters at the bottom of the file would be sufficient.

Reply:
The default is 1 thread-- the same code is run in a separate thread, then joined. I could make another function at the bottom, but it would be nearly identical to this.